src/BusinessLayer/DataPopulators/JsonDefines.h

```diff
@@ -216,12 +216,12 @@ namespace JsonFormat
     const QString AUXBMS_STROBEBMSLIGHT = "StrobeBmsLight";
     const QString AUXBMS_ALLOWCHARGE = "AllowCharge";
 
-    const QString AUXBMS_CHARGETRIP_HIGHVOLT = "ChargeTripDueToHighCellVoltage";
-    const QString AUXBMS_CHARGETRIP_HIGHTEMPCURR = "ChargeTripDueToHighTemperatureAndCurrent";
-    const QString AUXBMS_CHARGETRIP_PACKCURR = "ChargeTripDueToPackCurrent";
-    const QString AUXBMS_DISCHARGETRIP_PACKCURR = "DischargeTripDueToPackCurrent";
-    const QString AUXBMS_DISCHARGETRIP_LOWVOLT = "DischargeTripDueToLowCellVoltage";
-    const QString AUXBMS_DISCHARGETRIP_HIGHTEMPCURR = "DischargeTripDueToHighTemperatureAndCurrent";
+    const QString AUXBMS_CHARGETRIP_HIGHVOLTAGE = "ChargeTripDueToHighCellVoltage";
```

Comment:
@bill-luu Should this be `CHARGE_TRIP` instead?

Reply:
Yee sure